packages/beacon-node/src/network/discv5/worker.ts

```diff
@@ -56,7 +64,7 @@ const discv5 = Discv5.create({
     ip6: workerData.bindAddrs.ip6 ? multiaddr(workerData.bindAddrs.ip6) : undefined,
   },
   config: workerData.config,
-  metricsRegistry,
+  metricsRegistry: metricsRegistry as IDiscv5CreateOptions["metricsRegistry"],
```

Comment:
discv5 metrics require collect function to be defined, see [discv5/src/metrics.ts#L17](https://github.com/ChainSafe/discv5/blob/28eb9d2a2f1c6297a9ef677fcdb899a89d0f3af9/packages/discv5/src/metrics.ts#L17). Can clean this up once we reuse same metric types across all packages